build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -97,7 +79,7 @@
         name: amq7-interconnect-operator
         namespace: "{{ namespace }}"
       spec:
-        channel: 1.2.0
+        channel: 1.10.x
```

Comment:
I'm still seeing the docs mention 1.2 [1] but I can confirm that the cluster I'm running says 1.10.1

[1] https://github.com/infrawatch/documentation/blame/master/doc-Service-Telemetry-Framework/modules/proc_subscribing-to-the-service-telemetry-operator.adoc#L79

Reply:
Thanks! I'm going to do an updated copy/pasta of the output since that hasn't been done in a while (since we did the initial STF 1.3 release). I've been meaning to do this in a couple of other spots as well.

Reply:
I have created https://issues.redhat.com/browse/STF-587 to track the updates for both STF 1.3 and 1.4 documentation. I will deal with that separately of this issue.